ample/util/options_processor.py

```diff
@@ -381,6 +325,16 @@ def process_options(optd):
         msg = "Cannot find theseus executable: {0}".format(optd['theseus_exe'])
         exit_util.exit_error(msg)
 
+    # SCRWL - we always check for SCRWL as if we are processing QUARK models we want to add sidechains to them
+    if not optd['scwrl_exe']:
+        optd['scwrl_exe'] = 'Scwrl4' + ample_util.EXE_EXT
```

Comment:
Maybe make absolute path to binary by setting `os.path.join(os.environ['CCP4'], 'bin', 'Scwrl4' + ample_util.EXE_EXT)`